getting-started/cloudflare-workers.md

```diff
@@ -273,6 +273,14 @@ app.put('/upload/:key', async (c, next) => {
 This is the only case for Module Worker mode.
 If you want to use Variables or Secret Variables in Middleware, for example, "username" or "password" in Basic Authentication Middleware, you need to write like the following.
 
+First add this import at the top of `index.tsx`:
```

Comment:
The description is not needed. Just write the code like the following:

```ts
import { basicAuth } from 'hono/basic-auth'

// ...

app.use('/auth/*', async (c, next) => {
  const auth = basicAuth({
    username: c.env.USERNAME,
    password: c.env.PASSWORD,
  })
  return auth(c, next)
})
```

Reply:
fixed